core/templates/dev/head/services/context.service.ts

```diff
@@ -225,6 +225,17 @@ export class ContextService {
       this.getPageContext() ===
         ServicesConstants.PAGE_CONTEXT.EXPLORATION_EDITOR);
   }
+
+  // Following method helps to know whether RTE components requiring fullscreen
```

Comment:
I don't understand what fullscreen has to do with this ...?

Reply:
I'll change this comment